app/src/main/java/org/simple/clinic/patient/PatientAddress.kt

```diff
@@ -36,6 +36,17 @@ data class PatientAddress(
 
     val deletedAt: Instant?
 ) : Parcelable {
+
+  val addressFieldsString: String
+    get() = listOf(
+        streetAddress,
+        colonyOrVillage,
+        district,
+        state,
+        zone
+    ).filterNot { it.isNullOrBlank() }
+        .joinToString()
```

Comment:
I don't like how this is formatted, but it seems like our current settings format it in this way. Maybe we can look at changing our settings later to format it better.